### PR TITLE
Arianna

### PR DESCRIPTION
**Pull request by Thra11**
###### Description of changes

Package a new plasma / plasma mobile epub reader application, Arianna. Requries kirigami-addons 0.8.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/arianna/default.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+, qtbase
+, qtquickcontrols2
+, qtwebsockets
+, qtwebengine
+}:
+
+mkDerivation rec {
+  pname = "arianna";
+
+  version = "1.0.0";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/arianna/default.nix:23]:
```suggestion
  version = "1.0.0";

```

File: pkgs/applications/misc/arianna/default.nix [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  src = fetchFromGitLab {
+    domain = "invent.kde.org";
+    owner = "graphics";
+    repo = pname;
```
Comment (AndersonTorres) [inline pkgs/applications/misc/arianna/default.nix:27]:
```suggestion
    repo = "arianna";
```

File: pkgs/development/libraries/kirigami-addons/default.nix [hunk 3]
```diff
@@ -12,14 +12,14 @@
...
-  version = "0.7.2";
+  version = "0.8.0";
 
   src = fetchFromGitLab {
     domain = "invent.kde.org";
     owner = "libraries";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-2s9ShwYd2hOb18WRA2nLst6Q4UBHvFL+g7Grpjclz9I=";
+    sha256 = "sha256-ObbpM1gVVFhOIHOla5YS8YYe+JoPgdZ8kJ356wLTJq4=";
```
Comment (AndersonTorres) [inline pkgs/development/libraries/kirigami-addons/default.nix:22]:
```suggestion
    hash = "sha256-ObbpM1gVVFhOIHOla5YS8YYe+JoPgdZ8kJ356wLTJq4=";
```

File: pkgs/top-level/all-packages.nix [hunk 4]
```diff
@@ -28916,6 +28916,8 @@ with pkgs;
 
   argo-rollouts = callPackage ../applications/networking/cluster/argo-rollouts { };
 
+  arianna = libsForQt5.callPackage ../applications/misc/arianna { };
```
Comment (AndersonTorres) [inline pkgs/top-level/all-packages.nix:28919]: Does it run on Qt6?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Review by AndersonTorres — APPROVED** [review]

**Comment by AndersonTorres** [comment]
Ping me 2 days and I'll merge.

**Comment by Thra11** [comment]
> Ping me 2 days and I'll merge.

Ping, @AndersonTorres. Thanks!

**Pull request merged**